### PR TITLE
Upgrade Rook to 1.11.11

### PR DESCRIPTION
**Pull request by simu**
Replaces #115 

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

File: docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc [hunk 1]
```diff
@@ -4,9 +4,9 @@ include::partial$runbooks/contribution_note.adoc[]
 
 == icon:glasses[] Overview
 
-The MGR/Prometheus module is unreachable.
-This could mean that the module has been disabled or the MGR itself is down.
-Without the MGR/Prometheus module metrics and alerts will no longer function.
+The mgr/prometheus module at {{ $labels.instance }} is unreachable.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc:7]: Having alert templates in the runbook is pointless.

```suggestion
The mgr/prometheus module is unreachable.
```

File: docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc [hunk 2]
```diff
@@ -4,9 +4,9 @@ include::partial$runbooks/contribution_note.adoc[]
...
+This could mean that the module has been disabled or the mgr daemon itself is down.
+Without the mgr/prometheus module metrics and alerts will no longer function.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc:9]: IIRC I explicitly capitalized and prettified these two sentences in 1.10

File: docs/modules/ROOT/pages/runbooks/CephMonDiskspaceLow.adoc [hunk 3]
```diff
@@ -6,11 +6,11 @@ include::partial$runbooks/contribution_note.adoc[]
 
 The space available to a monitor's store is approaching full (>70% is the default).
 You should increase the space available to the monitor(s).
-The monitor's store can be found at `/var/lib/rook/<MON_NAME>/data` on the host.
-
+The default directory is `/var/lib/ceph/mon-*/data/store.db` on traditional deployments, and `/var/lib/rook/mon-*/data/store.db` on the mon pod's worker node for rook.
 Look for old, rotated versions of `*.log` and `MANIFEST*`.
 IMPORTANT: Don't touch any `*.sst` files.
 Also check any other directories under `/var/lib/rook` and other directories on the same filesystem, often `/var/log` and `/var/tmp` are culprits.
+Your monitor hosts are; {{- range query "ceph_mon_metadata"}} - {{ .Labels.hostname }} {{- end }}
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/CephMonDiskspaceLow.adoc:13]: Useless for the runbook

```suggestion
```

File: docs/modules/ROOT/pages/runbooks/CephNodeDiskspaceWarning.adoc [hunk 4]
```diff
@@ -4,10 +4,9 @@ include::partial$runbooks/contribution_note.adoc[]
...
-This alert is triggered when the disk space used by a Storage Node will be full in less than 5 days assuming the average fill-up rate of the past 48 hours.
-You should increase the space available to the node.
-The default location for the store sits under /var/lib/rook/ as a `hostPath` volume.
+Mountpoint {{ $labels.mountpoint }} on {{ $labels.nodename }} will be full in less than 5 days based on the 48 hour trailing fill rate.
 
 == icon:bug[] Steps for debugging
 
-include::partial$runbooks/check_mon_low_disk.adoc[]
+// Add detailed steps to debug and resolve the issue
+
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/CephNodeDiskspaceWarning.adoc:12]: Please revert all of this

File: docs/modules/ROOT/pages/runbooks/CephNodeInconsistentMTU.adoc [hunk 5]
```diff
@@ -4,9 +4,8 @@ include::partial$runbooks/contribution_note.adoc[]
 
 == icon:glasses[] Overview
 
-Node has a different MTU size than the median value on device.
+Node {{ $labels.instance }} has a different MTU size ({{ $value }}) than the median of devices named {{ $labels.device }}.
 
 == icon:bug[] Steps for debugging
 
 // Add detailed steps to debug and resolve the issue
-
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/CephNodeInconsistentMTU.adoc:12]: Revert

**Review by simu — COMMENTED** [review]
Please go over the runbooks again. The PR currently significantly reduces runbook quality:

Overall guidelines:
* Don't blindly commit the output of the runbook generator
* Label templates have no place in the runbook
* Compare changes carefully and think about what you'd like to see when you click on the runbook link in an alert, it's totally ok to rewrite the alert message to go more into the "why" of the alert instead of replicating the "what" from the alert description.

File: docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc (hunk 1 above)
Comment (DebakelOrakel) [reply docs/modules/ROOT/pages/runbooks/CephMgrPrometheusModuleInactive.adoc:7]: True, somehow i was in PrometheuRule mode 🤷

**Review by simu — COMMENTED** [review]
Changes mostly LGTM now, one remaining runbook fix inline.

Note that I can't explicitly approve the PR since I originally created it :see_no_evil:

**Review by DebakelOrakel — APPROVED** [review]

**Pull request merged**